site/assets/scss/_component-examples.scss

```diff
@@ -394,3 +394,8 @@
 
   // Boosted mod: no `.btn-clipboard`
 }
+
+.focused {
```

Comment:
I don't think this class is ever used. I'll try to confirm it and fix it upstream.